packages/paste-core/components/tooltip/src/index.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, BoxProps, safelySpreadBoxProps} from '@twilio-paste/box';
+import {StyledBase} from '@twilio-paste/theme';
+import {Text} from '@twilio-paste/text';
+import {
+  TooltipPrimitiveInitialState,
+  useTooltipPrimitiveState,
+  TooltipPrimitive,
+  TooltipPrimitiveReference,
+} from '@twilio-paste/tooltip-primitive';
+import {TooltipArrow} from './TooltipArrow';
+
+const StyledTooltip = React.forwardRef<HTMLDivElement, BoxProps>(({style, ...props}, ref) => {
+  return (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      backgroundColor="colorBackgroundBody"
+      borderColor="colorBorderLight"
+      borderRadius="borderRadius20"
+      borderStyle="solid"
+      borderWidth="borderWidth10"
+      boxShadow="shadowCard"
+      maxWidth="size30"
+      padding="space40"
+      paddingBottom="space30"
+      paddingTop="space30"
+      zIndex="zIndex10"
+      _focus={{outline: 'none'}}
+      style={style}
+      ref={ref}
+    />
+  );
+});
+
+export interface TooltipProps extends TooltipPrimitiveInitialState {
+  children: NonNullable<React.ReactElement>;
+  text: string;
+}
+
+const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(({children, text, ...props}, ref) => {
+  const tooltip = useTooltipPrimitiveState({...props});
+  return (
+    <>
+      {React.Children.only(
+        <TooltipPrimitiveReference {...tooltip} ref={ref} {...children.props}>
```

Comment:
Just curious, why are we passing `children.props` to the PrimitiveReference?

Reply:
Actually that was left over from an example on the reakit site. I don't think we need to them now. The reference just needs to clone the child.